### DISCUSSION TITLE
lv2 and clap support

### DISCUSSION THREAD

**Discussion by dreamer-design** [falkTX/Carla#1951] [Q&A]
i can't get Carla to load any lv2 files. i have the examples in the local/share/lib64/lv2 directory and have added to the path in config. I've also tried navigating to where the files are and they appear as folders and nothing to load. how do lv2 files work? 

also Carla is listed as supporting clap on there website and there's some code for it on the git but 2.5.9 doesn't have any support? is it wip?

**Comment by ColinKinloch** [comment]
The current release (2.5.10) doesn't have clap support. There is code on that branch for enumerating clap plugins but it doesn't appear to have the UI elements wired up.
The `main` branch has clap support which seems to work, though I only did some light testing with the Cardinal plugin.
